odd-platform-ui/src/components/Management/CollectorsList/CollectorItem/CollectorItem.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Grid, Typography } from '@mui/material';
+import {
+  Collector,
+  CollectorApiDeleteCollectorRequest,
+} from 'generated-sources';
+import LabeledInfoItem from 'components/shared/LabeledInfoItem/LabeledInfoItem';
+import ConfirmationDialog from 'components/shared/ConfirmationDialog/ConfirmationDialog';
+import EditIcon from 'components/shared/Icons/EditIcon';
+import DeleteIcon from 'components/shared/Icons/DeleteIcon';
+import AppButton from 'components/shared/AppButton/AppButton';
+import AppTooltip from 'components/shared/AppTooltip/AppTooltip';
+import CollectorFormDialogContainer from 'components/Management/CollectorsList/CollectorForm/CollectorFormContainer';
+import CollectorItemTokenContainer from './CollectorItemToken/CollectorItemTokenContainer';
+import * as S from './CollectorItemStyles';
+
+interface CollectorItemProps {
+  collector: Collector;
+  deleteCollector: (
+    params: CollectorApiDeleteCollectorRequest
+  ) => Promise<void>;
+}
+
+const CollectorItem: React.FC<CollectorItemProps> = ({
+  collector,
+  deleteCollector,
+}) => {
+  const onDelete = React.useCallback(
+    () => deleteCollector({ collectorId: collector.id }),
+    [collector]
+  );
+
+  return (
+    <S.Container elevation={0}>
+      <Grid container alignItems="flex-start" spacing={2}>
+        <Grid item xs={8}>
+          <Typography variant="h4" title={collector.name}>
+            {collector.name}
+          </Typography>
+        </Grid>
+        <S.ActionsContainer item sm={4}>
+          <CollectorFormDialogContainer
+            collector={collector}
+            btnCreateEl={
+              <AppButton
+                size="medium"
+                color="primaryLight"
+                startIcon={<EditIcon />}
+                sx={{ mr: 1 }}
+              >
+                Edit
+              </AppButton>
+            }
+          />
+          <ConfirmationDialog
+            actionTitle="Are you sure you want to delete this datasource?"
+            actionName="Delete"
+            actionText={
+              <Typography variant="subtitle1">
+                Delete &quot;{collector.name}&quot; datasource?
```

Comment:
Please use the naming associated with the collector

Reply:
done